docs/www/shadow-indexing.md

```diff
@@ -0,0 +1,401 @@
+
+It's all implemented with [minio](https://min.io/) so it should work with any cloud provider. We officially support Amazon's S3 and Google Cloud Storage. 
+
+You just have to configure your cloud credentials and we'll do the hard work for you. 
+
+With **Shadow Indexing**, Redpanda eliminates the need for disaster recovery by integrating with these infinitely scalable storage systems. It will push the old segments onto the cloud and fetch them when clients need them, with almost no risk to data loss.  
+
+On the surface, this sounds like the obvious solution in a cloud native world, but what it does for enterprises is that it unifies the way you access and manage both historical and real-time data without changing a single line of code in your application!
+
+We call this technology Shadow Indexing because it doesn’t keep the historical data on local disks, instead it keeps a location reference of how and where to fetch the data should a Kafka-API client request it. 
+
+What enterprises are the most excited about, is the fact that you can spin up a new Redpanda cluster and replay petabytes of data to retrain your machine learning model without affecting your production cluster or changing any code. 
+
+In a way, this presents a true disaggregation of compute and store for streaming data.
+
+## How it works
+
+When you create a topic with Redpanda, it automatically toggles the flag cloud_storage_enabled to integrate your data with the cloud. If you have your credentials set, then as soon as possible it will be pushed onto the object store of your choosing.
+
+We use Kafka's object `LastStableOffset(LSO)` to determine which data should be pushed into the cloud. Therefore, as soon as possible, the data is pushed into the object storage and subsequently evicted from the local machine.
+
+Your data is always uploaded to the remote storage first and only after that it can be deleted based on your retention policy. 
+
+Deletion can be done based on time (remove everything older than 2w, for example) or space (keep, say, 20GB of data per partition). 
+
+So when the deletion (space or time based) is enabled redpanda will guarantee that the log segments are deleted only if they're already uploaded. 
+
+So there is a guarantee that the data is always available locally or remotely.
+
+Please note that it’s not possible to provide outside data directly from the object store to Redpanda. You have to process your data through Redpanda for it to be accessible. After this, you can enjoy all the features listed here. 
+
+## How to get started
+
+It is very simple to activate Shadow Indexing in Redpanda. 
+
+You have to set this configuration in the topic
+
+
+```bash
+rpk topic create <topic-name> -c x-redpanda-shadowindex=true
+```
+
+```bash
+rpk topic alter-config <topic-name> --set x-redpanda-shadowindex=true
```

Comment:
this command enables shadow indexing on a topic that already exists, but shadow indexing is not enabled on a topic

```
rpk topic alter-config <topic-name> --set x-redpanda-remote-read=true --set x-redpanda-remote-write=true
```

Reply:
addressed